src/sql/workbench/browser/modelComponents/inputbox.component.ts

```diff
@@ -36,6 +36,7 @@ export default class InputBoxComponent extends ComponentBase implements ICompone
 	@Input() modelStore: IModelStore;
 	private _input: InputBox;
 	private _textAreaInput: InputBox;
+	private _defaultAriaLabel: string;
```

Comment:
Seems like this.defaultAriaLabel is always this.arialable - right?

just for my understanding - why cant we use this.arialable to do the if checks directly? I am saying that because that way... we can be sure even if somehow arialabel is set directly and not through Setproperties() and validate() was called - it will still have the right value.

if we mush do it in a separate variable... I would recommend setting this also as part of "public set ariaLabel(newValue: string)"

Reply:
Because we update aria-label with error messages, we need a variable somewhere to remember what the aria-label value used to be. setAriaLabel() will update this.ariaLabel and there's no functionality in the code that recalls the prior state of aria-label. This is why I created a separate variable.

I don't believe _defaultAriaLabel should ever need to be modified publicly--but please let me know your thoughts. Thanks Udeesha.